### DISCUSSION TITLE
HELP

### DISCUSSION THREAD

**Discussion by eliezerkif** [opensourcepos/opensourcepos#4000] [General]
![ospos](https://github.com/opensourcepos/opensourcepos/assets/170686651/8e36bfd2-fc2b-4546-a334-26506d99fd19)

what can i do?

**Comment by iranpayne** [comment]
Greetings, I have worked on previous versions, but I have not been able to get this version to work. If this ospos project has anyone managed to install it and can help me?

**Comment by odiea** [comment]
php 7.4 should work
